core/domain/feedback_jobs_continuous.py

```diff
@@ -44,15 +46,31 @@ class FeedbackAnalyticsAggregator(jobs.BaseContinuousComputationManager):
 
     @classmethod
     def get_event_types_listened_to(cls):
+        """Get the event types that this class subscribe to.
+
+        Returns:
+               list. List of event types that this class subscribe to.
```

Comment:
Use 4-space indentation, not 8.

Specify the type of the item in the list -- e.g. list(str), list(int), etc. (you'll need to figure out what it is). Also subscribe --> is subscribed

Reply:
Done.